agent/wptdriver/wpt_test.h

```diff
@@ -225,6 +225,11 @@ class WptTest {
   void      BuildScript();
   CAtlList<ScriptCommand> _script_commands;
 
+  //Timeout information
+  DWORD    _timeout_value_in_seconds;

```

Comment:
Given that there are so many obscure 'timeout's in WPT codebase, I'd suggest renaming this to _overall_timeout. Also, instead of having it in seconds, I'd suggest making it a millisecond value _overall_timeout_ms. That way, we avoid unnecessary multiplication every time in TestState::IsDone(). What do you think?